testkit/src/main/scala/org/bitcoins/testkit/wallet/BitcoinSWalletTest.scala

```diff
@@ -293,6 +303,22 @@ object BitcoinSWalletTest extends WalletLogger {
       chainQueryApi = chainQueryApi)(config, ec)() // get the standard config
   }
 
+  /** This wallet should have a total of 6 bitcoin in it
+    * spread across 3 utxos that have values 1, 2, 3 bitcoins */
+  case class FundedWallet(wallet: LockedWallet)
```

Comment:
Is this a strict requirement? If so should you add invariants?

Reply:
Especially since it appears that tests rely on this fact

Reply:
There is an invariant in `fundWallet` for this: https://github.com/bitcoin-s/bitcoin-s/pull/1010/files#diff-0fb6ac004fe1e550b7c13258d7d0706cR412